shell/common/shell.cc

```diff
@@ -899,6 +899,12 @@ void Shell::OnPlatformViewDestroyed() {
   // Overall, the longer term plan is to remove this implementation once
   // https://github.com/flutter/flutter/issues/96679 is fixed.
   rasterizer_->TeardownExternalViewEmbedder();
+
+  auto ui_task = [engine = engine_->GetWeakPtr()] {
+    if (engine) {
+      engine->NotifyDestroyed();
+    }
+  };
```

Comment:
It looks like this task never gets run?

Reply:
Fixed the code to actually ensure the task gets run.